itest/rust/src/object_tests/property_test.rs

```diff
@@ -239,6 +239,18 @@ struct CheckAllExports {
     #[export]
     normal: GString,
 
+    #[export_group(name = "test_group", prefix = "a_")]
+    #[export]
+    a_grouped: i64,
+
+    #[export_subgroup(name = "some group")]
+    #[export]
+    subgrouped: i64,
+
+    #[export_subgroup]
+    #[export]
+    ungrouped: i64,
```

Comment:
I feel like after `a_grouped`, there should be another field with just `#[export]`. Otherwise we don't test that `#[export_group]` applies to multiple subsequent fields.

Reply:
Makes sense, I also update `guaranteed_ordering` test.